documentation/modules/ROOT/pages/kubernetes.adoc

```diff
@@ -33,10 +50,10 @@ quarkus.datasource.db-kind=h2
 quarkus.hibernate-orm.database.generation=drop-and-create
 quarkus.hibernate-orm.sql-load-script=import.sql
 
-quarkus.container-image.registry=quay.io # <1>
```

Comment:
The whitespace causes issues when you copy/paste (the trailing blank space is copied to the application properties files which then causes the registry (and other values) to not be defined correctly.